staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -30,10 +30,10 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apiserver/pkg/util/trie"
+	"k8s.io/kube-openapi/pkg/util"
```

Comment:
It feels odd to me to share utils between repos if they are not in k8s.io/util. But I don't feel strongly about this one.